sys/include/net/nanocoap.h

```diff
@@ -762,6 +802,91 @@ static inline ssize_t coap_get_location_query(const coap_pkt_t *pkt,
                                target, max_len, '&');
 }
 
+/**
+ * @brief Initialize a block2 slicer struct
+ *
+ * This function initializes a block2 response and adds the block2 option to
+ * the packet
+ *
+ * Caller must ensure that this function is called after all required
+ * preceding options are added
+ *
+ * @param[in]   buf         buffer to write to
+ * @param[in]   lastonum    last option number (must be <27)
+ * @param[in]   pkt         packet to work on
+ * @param[out]  slicer      Preallocated slicer struct to fill
+ *
+ * @returns     amount of bytes written to @p buf
+ */
+void coap_block2_init(coap_pkt_t *pkt, coap_block_slicer_t *slicer);
```

Comment:
Documentation out of sync with declaration. I fixed in [b6b452d](https://github.com/bergzand/RIOT/pull/3/commits/b6b452d6af2d14d78b09a04749d7d8bfee6b7213).